tutorials/create-events-dataset.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: page
+title: Creating the Events Dataset
```

Comment:
I think this tutorial needs more context as well. Maybe there should be an overall tutorial page that outlines the example goal, or maybe it can be included at the start of the tutorials. Otherwise, it isn't clear why we're creating "the" events dataset or what this is used for.